py3status/modules/xrandr.py

```diff
@@ -399,11 +402,15 @@ def xrandr(self):
         self._set_available_combinations()
         self._choose_what_to_display()
 
-        if self.fixed_width is True:
-            output = self._center(self.displayed)
+        if (len(self.available_combinations) < 2 and
+                self.hide_if_single_combination):
+            full_text = ''
```

Comment:
You introduce `full_text = ''` here. I think it's okay if we skip that and go straight to `self.py3.safe_format(self.format)` + `[{output}]` instead. We add missing placeholders to work with powerful formatter stuffs in 3.6 or later.